src/main/resources/messages/UTBot.properties

```diff
@@ -0,0 +1,4 @@
+name=UTBot plugin
+projectConfigure.configure=Configure project
```

Comment:
How does it correspond to other actions? Can it be put into src/main/resources/META-INF/plugin.xml as well?

Reply:
This file is just a collection of string properties, so you don't put user messages in code. No these properties can't be put into plugin.xml